code/modules/mod/mod_theme.dm

```diff
@@ -0,0 +1,1316 @@
+/// Global proc that sets up all MOD themes as singletons in a list and returns it.
+/proc/setup_mod_themes()
+	. = list()
+	for(var/path in typesof(/datum/mod_theme))
+		var/datum/mod_theme/new_theme = new path()
+		.[path] = new_theme
+
+/// MODsuit theme, instanced once and then used by MODsuits to grab various statistics.
+/datum/mod_theme
+	/// Theme name for the MOD.
+	var/name = "standard"
+	/// Description added to the MOD.
+	var/desc = "A civilian class suit by Nakamura Engineering, doesn't offer much other than slightly quicker movement."
```

Comment:
We adding a company named nakamura?

Reply:
Right, I did want to look into the company names, thought that one was TG only. Will get a more accurate name.

Reply:
Removed